django/apps/blue_management/blue_mgnt/static/js/billing.js

```diff
@@ -58,14 +58,26 @@
 
     var COUPON_CACHE = {};
 
+    var monthly_cost = 5;
+    var yearly_cost = 60;
+    $.ajax("/billing/info", {
+        type: "GET",
+        // FIXME: "async: false" deprecated but simplest solution for now
+        async: false,
+        success: function(data, status, xhr) {
+                monthly_cost = "monthly_cost" in data ? data["monthly_cost"] : 5;
+                yearly_cost = "yearly_cost" in data ? data["yearly_cost"] : 60;
+            },
+    });
```

Comment:
What happens if this ajax fails? Could the page potentially show the user the wrong price?